controllers_v2/invoice.ctrl.go

```diff
@@ -134,6 +134,7 @@ type AddInvoiceResponseBody struct {
 	PaymentHash    string    `json:"payment_hash"`
 	PaymentRequest string    `json:"payment_request"`
 	ExpiresAt      time.Time `json:"expires_at"`
+	CreatedAt      time.Time `json:"created_at"`
```

Comment:
can we return this as timestamp? otherwise we have to deal with timezones?

Reply:
We cannot change expires_at - it needs to be backward compatible. I think created_at should be consistent here.

We could add 2 new fields for timestamp format. What do you think? @bumi @kiwiidb

Reply:
CC @jankoegel